src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Kindly remove this `eslint-disable` statement. Eslint helps us write JavaScript codes that are written to industry standards, are consistent and potentially avoid bugs. Using the `eslint-disable` statement defeats this purpose. I would advise you to export the store as _**default**_. 💻 😎